osmnames/import_osm/set_names.sql

```diff
@@ -0,0 +1,52 @@
+-- If the 'name' is empty or null, get name in another available language
+-- Order of priority for 'name' is: default name, en, fr, de, es, ru, zh
+DROP FUNCTION IF EXISTS get_name(HSTORE);
+CREATE FUNCTION get_name(all_tags HSTORE) RETURNS TEXT AS $$
+  SELECT COALESCE(all_tags -> 'name:en',
+                  all_tags -> 'name:fr',
+                  all_tags -> 'name:de',
+                  all_tags -> 'name:es',
+                  all_tags -> 'name:ru',
+                  all_tags -> 'name:zh');
+$$ LANGUAGE 'sql' IMMUTABLE;
+
+
+
+CREATE OR REPLACE FUNCTION array_distinct(anyarray)
```

Comment:
with the proposal for get_alternative_names, you can remove this function :smiley: